.gitlab-ci.yml

```diff
@@ -1891,74 +2016,151 @@ twistlock_scan-7:
 
 dev_branch_docker_hub:
```

Comment:
I'm curious: why are we keeping both the `dev_branch_docker_hub` and `dev_branch_multiarch_docker_hub` jobs? To me it seems like the second one does everything that the first one does, and then some more

Reply:
Since the ARM build is pretty long, this makes it possible to use an amd64 image from docker hub quicker if you don't care about the ARM part.